### PR TITLE
Organisations - Detach policy, and asserts it actually happens

### PR DESCRIPTION
**Pull request by bblommers**
As pointed out by @dpowley in #3278:
Detaching a policy was broken, and the existing test didn't actually assert that the detachment was successful.

### CODE REVIEW

**Review by dpowley — APPROVED** [review]
LGTM - Thanks for the quick fix

**Comment by dpowley** [comment]
@bblommers What timeline can I expect for a release with this bug fix included? We would use this on day one 😄

**Comment by bblommers** [comment]
@dpowley Probably one or two days after approval from a maintainer! No timeline on when the approval comes in though

**Comment by dpowley** [comment]
> @dpowley Probably one or two days after approval from a maintainer! No timeline on when the approval comes in though

@bblommers I saw there have been a few releases since this one opened up. Any chance we can get this fix out the door soon? It also causes issues with testing DeletePolicy. If the policy is attached to a target, you can't delete the policy without first detaching it from the targets. And there is no way to detach the policy currently (this PR fixes the detachment bug).

**Pull request merged**
